app/scripts/services/ruleEngine.js

```diff
@@ -7,10 +7,50 @@
  * # Rule Engine service
  * Service to interface with the HMDA Rule Engine
  */
-module.exports = function () {
+module.exports = function (HMDAEngine) {
 
-    // Get a list of the Fiscal Years in use by the HMDA Rule Engine
+    var filename;
+
+    /**
+     * Get a list of the Fiscal Years in use by the HMDA Rule Engine
+     * TODO: Expose the get years method in the hmda-spec to the hmda-rule-engine
```

Comment:
This TODO is done.. Might want to remove the `getFiscalYears()`, or passthrough to `HMDAEngine.getValidYears()`